src/rdrand.c

```diff
@@ -31,33 +31,85 @@
 #include "rdtime.h"
 #include "tinycthread.h"
 #include "rdmurmur2.h"
+#ifndef _WIN32
```

Comment:
Can we write unit tests for this file?

Reply:
Yes I agree but I think its good to have unit tests for this module as well. Maybe write a TODO or create a issue for the same.